lib/workers/repository/init/index.ts

```diff
@@ -11,21 +11,32 @@ import { mergeRenovateConfig } from './config';
 import { detectSemanticCommits } from './semantic';
 import { detectVulnerabilityAlerts } from './vulnerability';
 
-export async function initRepo(input: RenovateConfig): Promise<RenovateConfig> {
+function initializeConfig(config: RenovateConfig): RenovateConfig {
+  return { ...config, errors: [], warnings: [], branchList: [] };
+}
+
+async function initializeCaches(config: RenovateConfig): Promise<void> {
   memCache.init();
-  await repositoryCache.initialize(input);
-  let config: RenovateConfig = {
-    ...input,
-    errors: [],
-    warnings: [],
-    branchList: [],
-  };
-  config = await initApis(config);
+  await repositoryCache.initialize(config);
+}
+
+async function getRepoConfig(config_: RenovateConfig): Promise<RenovateConfig> {
+  let config = { ...config_ };
```

Comment:
Only flat copy? Is that necessary?

Reply:
Just trying to avoid the lint warnings. This is intended as a do-nothing refactoring so that the lazy git changes I have then becomes just a few changed lines.